testing/regress/ecl/signingPass.ecl

```diff
@@ -0,0 +1,49 @@
+IMPORT STD;
```

Comment:
This needs a matching testing/regress/ecl/key/signingPass.xml

Reply:
Added key file.